ide/xml.text/apichanges.xml

```diff
@@ -91,9 +91,16 @@ is the proper place.
             <compatibility binary="compatible" source="incompatible" semantic="compatible" addition="yes" deprecation="yes" deletion="yes"/>
             <description>
                 <p>
-                    The xml.text.syntax API uses the old and long deprecated lexer. The API is now moved to a new compatibility-only 
+                    The xml.text.syntax API uses the old and long deprecated lexer. The API was moved to a new compatibility-only 
                     <code>xml.text.deprecated90</code> module and a new API in <code>org.netbeans.modules.xml.text.api.dom</code> package is offered.               
```

Comment:
Prehaps the mention of the deprecated90 (should have been obsolete90) can be removed as well.